src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int FIGURES_QUANTITY = 10;
+
+    public static void main(String[] args) {
+        FigureSupplier supplier = new FigureSupplier();
+        for (int i = 0; i < FIGURES_QUANTITY; i++) {
+            supplier.getRandomFigure().draw();

```

Comment:
According the task description
In the main() method we need to create an array of figures (the size of array can be 3 or 6, it doesn't matter). 
1) The first half of figures in this array should be generated with random parameters.
2) The other half of figures should have the same, default parameters.
The 1-st part is implemented, please implement the 2nd part too.